src/components/ReadWrite.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { FirestoreCollection, withFirestore } from 'react-firestore';
+
+function ReadWrite(props) {
+  const { firestore } = props;
+  const [enteredValue, setEnteredValue] = useState();
+
+  const addMsg = e => {
+    e.preventDefault();
+    firestore.collection('example').add({
+      msg: enteredValue,
+    });
+  };
+
+  return (
+    <div>
+      <form>
+        <input
+          type="text"
+          name="msg"
+          placeholder="enter your message here..."
+          onChange={e => setEnteredValue(e.target.value)}
+        />
```

Comment:
It looks like Accessibility Insights for Web need this to have a label attribute so that it meets accessibility standards.[ Download this extension here and give the FastPass option a run for more details.](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni) 

![](https://cdn.zappy.app/de9992e7b446378f941e048652184a67.png)

[Here are some more details](https://www.w3.org/WAI/tutorials/forms/labels/#:~:text=Provide%20labels%20to%20identify%20all,purpose%20of%20the%20form%20control.) on labeling your form controls!